packages/styles/page-header.css

```
@@ -0,0 +1,14 @@
+:root {
+  --page-header-overline-color: var(--gray-60);
+}
+
+.cauldron--theme-dark {
+  --page-header-overline-color: var(--accent-light);
+}
+
+.PageHeader > p {
```

See above suggestion regarding usage of the `p` tag.

```suggestion
.PageHeader__overline {
```